### DISCUSSION TITLE
I want to join the team,,, my username on github is EmadGr8

### DISCUSSION THREAD

**Discussion by EmadGr8** [ROM-builders/temporary#16795] [Q&A]
successful build
https://cirrus-ci.com/build/6189267452952576
device tree
https://github.com/EmadGr8/device_xiaomi_tulip
vendor tree
https://github.com/EmadGr8/vendor_xiaomi_tulip-1
kernel tree
https://github.com/EmadGr8/kernel_xiaomi_sdm660-MOCHA-fixed

**Comment by Apon77** [comment]
links are not available

**Reply by EmadGr8** [reply]
successful build
https://cirrus-ci.com/build/6627492497195008
device tree
https://github.com/EmadGr8/mas-device_xiaomi_tulip
vendor tree
https://github.com/EmadGr8/mas-vendor_xiaomi_tulip
kernel tree
https://github.com/EmadGr8/mas-kernel_xiaomi_sdm660

PS. sorry I renamed repos
new ones with more recent successful build above

**Reply by EmadGr8** [reply]
about #learn instructions
I am willing to learn more, but I don't have a PC which is why I am using cirrus-ci.com
I just need to join team to bypass server queue limit
thank you anyways man

**Reply by Apon77** [reply]
without following #learn you wont be added in team. please follow the process. thank you
